app/views/locales/index.html.erb

```diff
@@ -17,12 +17,14 @@
         <th scope="col">Name</th>
         <th scope="col">ISO code</th>
         <th scope="col">Primary locale</th>
+        <th scope="col">Allow the locale to be used for email</th>
```

Comment:
Your right, however the scope of this is to allow emails to me localised, not the interface of the archive and in general our views are not localised today so I didn't put that chunk of work in scope.